core/frontend/src/components/autopilot/EndpointCard.vue

```diff
@@ -179,7 +176,7 @@ export default Vue.extend({
         })
     },
     async toggleEndpointEnabled(): Promise<void> {
-      this.updated_endpoint.enabled = !this.updated_endpoint.enabled
+      await sleep(500)
```

Comment:
Wait, why this sleep is necessary ?

Reply:
It's just so the user can see the switch toggle animation 😄 
Without it, the spinning logo will kick in as soon as the user clicks the switch.

Reply:
It would be nice to have a comment so, this is not clear when reading the code.

Reply:
Yup. Adding.

Reply:
@rafaellehmkuhl still missing comment.